### PR TITLE
Use a patched version of uuid_enhanced that does not subclass typed_data classes

### PR DESCRIPTION
**Pull request by jason-simmons**
The typed_data classes are sealed in the latest Dart SDK
(see https://github.com/dart-lang/sdk/commit/3e9cdc56440bded76d8078a2567910236ad6f7b9)

### CODE REVIEW

**Comment by jason-simmons** [comment]
That was not intentional.  Regenerated the PR.

File: app_dart/uuid_enhanced/.cirrus.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+container:
```
Comment (CaseyHillers) [inline app_dart/uuid_enhanced/.cirrus.yml:1]: Cocoon has `tests.yaml` where we could add this as a standard Cocoon dart project (would be added to the existing LUCI builder)

**Review by CaseyHillers — APPROVED** [review]
Overall, this LGTM.

I'm hesitant to fork the package if https://github.com/flutter/cocoon/pull/1209 can unblock the Dart roll

**Comment by CaseyHillers** [comment]
@jason-simmons can we close this, or is the roll still blocked?

**Pull request closed without merging**